src/main/java/com/fasterxml/jackson/databind/deser/std/StringArrayDeserializer.java

```diff
@@ -40,14 +40,14 @@ public final class StringArrayDeserializer
     /**
      * Value serializer to use, if not the standard one (which is inlined)
      */
-    protected JsonDeserializer<String> _elementDeserializer;
+    private final JsonDeserializer<String> _elementDeserializer;
```

Comment:
This PR is targeted at branch 2.19 meaning a patch release. Patches should not contain changes to variable visibility. Users could have subclasses of Jackson classes and hiding the variables could break their subclasses.

Reply:
Since this is a `final class`, there should be no subclasses.
https://github.com/FasterXML/jackson-databind/blob/34ae9f52db2c1b55a81d746d01e39e1b6d074a51/src/main/java/com/fasterxml/jackson/databind/deser/std/StringArrayDeserializer.java#L27